src/components/MobileNavbar/MobileNavbar.js

```diff
@@ -39,7 +39,7 @@ const NAVBAR_LINKS = [
 ]
 
 const MENU_LINKS = [
-  { linkTo: '/sonar/my-signals', label: 'Alerts' },
+  { linkTo: '/sonar/my-alerts', label: 'Alerts' },
```

Comment:
I think we also start to avoid `sonar` naming. Maybe left just `/my-alerts` path?